### DISCUSSION TITLE
All deploys fail

### DISCUSSION THREAD

**Discussion by 3leonora** [alshedivat/al-folio#671] [Q&A]
Hi!

I'm in love with this template and I would really like a web page like this. Thanks for making it!

I've been following the guide thoroughly, but the deploy (or all the workflow actions) fail every time I commit. I have made sure that both `url` and `baseurl` are empty (I also tried setting url to my website url [https://3leonora.github.io](https://3leonora.github.io)).  What can I do?

**Comment by rohandebsarkar** [comment]
Do not keep the url empty.
Please post the GitHub action logs.
